src/packages/skeleton/skeleton.scss

```
@@ -39,50 +39,93 @@
     width: 100%;
     height: 100%;
     z-index: 1;
-    background: linear-gradient(
+    // 80px 高光带由 background-size 控制,元素本身占满整行,
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)